### PR TITLE
Added `Separator` documentation

### PR DESCRIPTION
**Pull request by hirotomoyamada**
## Description

Added `Separator` documentation

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

File: docs/scripts/component/props.ts
Comment (Itsuki54) [inline docs/scripts/component/props.ts:1]: Do I need to delete the 'divider' here?

Comment (hirotomoyamada) [reply docs/scripts/component/props.ts:1]: I worked on it while I was at it. I just replaced it and hope the build is successful.

Comment (Itsuki54) [reply docs/scripts/component/props.ts:1]: I'm sorry, my question was bad,
 don't I have to remove 'divider' here?

Comment (hirotomoyamada) [reply docs/scripts/component/props.ts:1]: I do not understand your question. Please tell me where in the source code you are referring to.

Comment (Itsuki54) [reply docs/scripts/component/props.ts:1]: sorry, I think line 106 is to need  delete

Comment (Itsuki54) [reply docs/scripts/component/props.ts:1]: I'm sorry.
I understand.
LGTM

**Review by Itsuki54 — APPROVED** [review]

**Pull request merged**